### DISCUSSION TITLE
Heat Schedule

### DISCUSSION THREAD

**Discussion by matthias811** [Yogui79/IntexPureSpa#25] [General]
In the video showing the mobile screen there is as well the functionality to setup a schedule for the heater like it is as well present on the remote control. In the code and mqtt topics I cannot find the right topic. Is this directly supported with the spa.ino or is a external scheduler (homecontrol?) being used?
By the way: I changed the mqtt calls to have retain flat set to true and added a enableLastWillMessage that is raised if the ESP connection looses its connection to the broker (mine is currently running on battery...). If someone is interested I can share the code

**Comment by djgel** [comment]
If I understood, you want to schedule the heating, but since you have it integrated in HA, you can already create an automation triggering the heating trigger you want!

**Reply by LEECHER1** [reply]
Yes, it is a Automation in HA. 

I have found a new/good way the schedul all function for the Spa. 

The Scheduler HACS plugin.
https://github.com/nielsfaber/scheduler-component

**Comment by Yogui79** [comment]
@matthias811 
please make a fork from the project and add your change and send me the link I will look if  its interresting and/or if I caninclude that as option in the project
